src/shiftedRootNormLhalfBinf.jl

```diff
@@ -52,25 +52,24 @@ function prox!(
   q::AbstractVector{R},
   σ::R,
 ) where {R <: Real, V0 <: AbstractVector{R}, V1 <: AbstractVector{R}, V2 <: AbstractVector{R}}
-  γλ = 2 * σ * ψ.λ
-  ϕ(z) = acos(γλ / 8 * (abs(z) / 3)^(-3 / 2) + 0im)
-  # p = 54^(1/3) * (γλ^(2/3)) / 4 # not necessary to compute
+  νλ = 2 * σ * ψ.λ
+  ϕ(z) = acos(νλ / 8 * (abs(z) / 3)^(-3 / 2) + 0im)
 
-  qbar = q + (ψ.xk .+ ψ.sj)
+  ψ.sol .= q .+ (ψ.xk .+ ψ.sj)
   RNorm(tt, l) = 0.5 / σ * (tt - q[l])^2 + ψ.λ * sqrt(abs(tt + ψ.sj[l] + ψ.xk[l]))
   t = zeros(4) #probably not smart to use arrays, but can change
   ft = similar(t)
```

Comment:
Can we eliminate these allocations?

Reply:
Would you rather me store x + s instead of q + x + s? I'm not sure I understand.

Reply:
I'm talking about t and ft, which are allocated afresh at each prox evaluation.

Reply:
Gotcha - I tried, lmk if it looks ok.